cli/admin/src/test/java/io/pravega/cli/admin/dataRecovery/DataRecoveryTest.java

```diff
@@ -1632,6 +1634,82 @@ public void testDurableLogInspectCommandException() throws Exception {
+            // Write events to the streams.
+            TestUtils.writeEvents(streamName, clientRunner.getClientFactory());
+        }
+        pravegaRunner.shutDownControllerRunner(); // Shut down the controller
+
+        // Flush all Tier 1 to LTS
+        ServiceBuilder.ComponentSetup componentSetup = new ServiceBuilder.ComponentSetup(pravegaRunner.getSegmentStoreRunner().getServiceBuilder());
+        for (int containerId = 0; containerId < containerCount; containerId++) {
+            componentSetup.getContainerRegistry().getContainer(containerId).flushToStorage(TIMEOUT).join();
+        }
+
+        pravegaRunner.shutDownSegmentStoreRunner(); // Shutdown SegmentStore
+        pravegaRunner.shutDownBookKeeperRunner(); // Shutdown BookKeeper & ZooKeeper
+
+        // start a new BookKeeper and ZooKeeper.
+        pravegaRunner.close();
+        LocalServiceStarter.PravegaRunner pravegaRunner2 = new LocalServiceStarter.PravegaRunner(bookieCount, containerCount);
+        pravegaRunner2.startBookKeeperRunner(instanceId++);
+
+        // set pravega properties for the test
+        STATE.set(new AdminCommandState());
+        Properties pravegaProperties = new Properties();
+        pravegaProperties.setProperty("pravegaservice.container.count", "1");
+        pravegaProperties.setProperty("pravegaservice.storage.impl.name", "FILESYSTEM");
+        pravegaProperties.setProperty("pravegaservice.storage.layout", "CHUNKED_STORAGE");
+        pravegaProperties.setProperty("filesystem.root", this.baseDir.getAbsolutePath());
+        pravegaProperties.setProperty("pravegaservice.zk.connect.uri", "localhost:" + pravegaRunner2.getBookKeeperRunner().getBkPort());
+        pravegaProperties.setProperty("bookkeeper.zk.connect.uri", "localhost:" + pravegaRunner2.getBookKeeperRunner().getBkPort());
+        pravegaProperties.setProperty("bookkeeper.ledger.path", pravegaRunner2.getBookKeeperRunner().getLedgerPath());
+        pravegaProperties.setProperty("bookkeeper.zk.metadata.path", pravegaRunner2.getBookKeeperRunner().getLogMetaNamespace());
+        pravegaProperties.setProperty("pravegaservice.clusterName", pravegaRunner2.getBookKeeperRunner().getBaseNamespace());
+        pravegaProperties.setProperty("writer.flush.attributes.threshold", "1");
+        STATE.get().getConfigBuilder().include(pravegaProperties);
+
+        // Copy Metadata and Storage Metadata chunks to separate directory
+        File metadataChunksDir = Files.createTempDirectory("TestLTSRecovery").toFile().getAbsoluteFile();
+        List<Path> filtered = null;
+        try (Stream<Path> pathStream = Files.list(Path.of(this.baseDir.getAbsolutePath() + "/_system/containers"))) {
+            filtered = pathStream.filter(path -> path.toFile().getName().startsWith("metadata_") || path.toFile().getName().startsWith("storage_metadata_"))
+                    .collect(Collectors.toList());
+        }
+        //copy to {metadataChunksDir}
```

Comment:
Cleanup code, in general: unnecessary spaces, comments not following style guidelines, etc.